Documentation/CRDs/ceph-nfs-crd.md

```diff
@@ -78,122 +78,37 @@ The `server` spec sets configuration for Rook-created NFS-Ganesha servers.
-  "security_label": true,
-  "protocols": [
-    4
-  ],
-  "transports": [
-    "TCP"
-  ],
-  "fsal": {
-    "name": "CEPH",
-    "user_id": "nfs.my-nfs.1",
-    "fs_name": "myfs"
-  },
-  "clients": []
-}
-```
-
-If you are done managing NFS exports and don't need the Ceph orchestrator module enabled for
-anything else, it may be preferable to disable the Rook and NFS mgr modules to free up a small
-amount of RAM in the Ceph mgr Pod.
-
-```console
-ceph orch set backend ""
-ceph mgr module disable rook
-```
-
-### Mounting exports
-
-Each CephNFS server has a unique Kubernetes Service. This is because NFS clients can't readily
-handle NFS failover. CephNFS services are named with the pattern
-`rook-ceph-nfs-<cephnfs-name>-<id>` `<id>` is a unique letter ID (e.g., a, b, c, etc.) for a given
-NFS server. For example, `rook-ceph-nfs-my-nfs-a`.
-
-For each NFS client, choose an NFS service to use for the connection. With NFS v4, you can mount an
-export by its path using a mount command like below. You can mount all exports at once by omitting
-the export path and leaving the directory as just `/`.
-
-```console
-mount -t nfs4 -o proto=tcp <nfs-service-address>:/<export-path> <mount-location>
-```
-
-## Exposing the NFS server outside of the Kubernetes cluster
+### Security
+
+The `security` spec sets security configuration for the NFS cluster.
+
+* `sssd`: SSSD enables integration with System Security Services Daemon (SSSD). See also:
+  [ID management via SSSD](../Storage-Configuration/NFS/nfs-security.md#id-management-via-sssd).
+  * `sidecar`: Specifying this configuration tells Rook to run SSSD in a sidecar alongside the NFS
+    server in each NFS pod.
+    * `image`: defines the container image that should be used for the SSSD sidecar.
+    * `sssdConfigFile`: defines where the SSSD configuration should be sourced from. The
+      config file will be placed into `/etc/sssd/sssd.conf`. If this is left empty, Rook will not
+      add the file. This allows you to inject your own, for example by using
+      [Vault's agent injector](https://www.vaultproject.io/docs/platform/k8s/injector).
+      * `volumeSource`: this is a standard Kubernetes
+        [VolumeSource](https://pkg.go.dev/k8s.io/api/core/v1#VolumeSource) like what is normally
+        used to configure Volumes for a Pod. For example, a ConfigMap, Secret, or HostPath.
+        There are two requirements for the source's content:
+        1. The config file must be mountable via `subPath: sssd.conf`. For example, in a ConfigMap,
+           the data item must be named `sssd.conf`, or `items` must be defined to select the key and
+           give it path `sssd.conf`.
+        2. The volume or config file must have mode 0600.
+    * `debugLevel`: sets the debug level for SSSD. If unset or `0`, Rook does nothing. Otherwise,
+      this may be a value between 1 and 10. See the
+      [SSSD docs](https://sssd.io/troubleshooting/basics.html#sssd-debug-logs) for more info.
```

Comment:
This is present on the NFS CRD doc. We could break some of the explanation for `sssdConfigFile`/`volumeSource` into a section if you think it's too long. It seems a bit long to me, but not quite so long that it needs its own section.